Analytics-CSharp/Segment/Analytics/Analytics.cs

```diff
@@ -130,32 +108,21 @@ public async Task<string> UserIdAsync()
         /// <returns><see cref="JsonObject"/> instance of Traits</returns>
         public JsonObject Traits()
```

Comment:
and here

Reply:
=> makes template unhappy, see other comment